mvp/src/test/java/com/hannesdorfmann/mosby3/mvp/conductor/delegate/MvpConductorLifecycleListenerTest.java

```diff
@@ -138,7 +141,7 @@
 
     // Detach
     lifecycleListener.preDestroyView(controller, view);
-    Mockito.verify(presenter, Mockito.times(1)).detachView(false);
+    Mockito.verify(presenter, Mockito.times(1)).detachView();
```

Comment:
Not sure if there is more to this particular test? any suggestions?

Reply:
yeah it needs some more testing of `presenter.destroy()` but I will take this over